specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -1798,6 +2706,122 @@
       },
       "x-ms-discriminator-value": "ApiKey"
     },
+    "CognitiveServiceEndpointDeploymentResourceProperties": {
+      "required": [
+        "model"
+      ],
+      "type": "object",
+      "properties": {
+        "model": {
+          "description": "Model used for the endpoint deployment.",
+          "$ref": "#/definitions/EndpointDeploymentModel"
+        },
+        "properties": {},
+        "raiPolicyName": {
+          "type": "string"
+        },
+        "sku": {
+          "$ref": "#/definitions/CognitiveServicesSku"
+        },
+        "versionUpgradeOption": {
+          "$ref": "#/definitions/DeploymentModelVersionUpgradeOption"
+        }
+      }
+    },
+    "CognitiveServicesSku": {
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "family": {
+          "type": "string"
+        },
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Don't duplicated top-level property names. Recommend skuName for the name of this property.